rotkehlchen/accounting/cost_basis/base.py

```diff
@@ -257,12 +260,14 @@ def inform_user_missing_acquisition(
             )
             return
 
-        self.msg_aggregator.add_error(
-            f'Not enough documented acquisitions found for {asset} before '
-            f'{self.timestamp_to_date(time)}. Only found acquisitions '
-            f'for {found_amount} {asset} and miss {missing_amount} {asset}.'
-            f'Let rotki know how you acquired it via a ledger action',
-        )
+        if missing_amount is not None:
```

Comment:
Hmm it makes sense if you look at the function declaration to do this check but if you see how the function is called, if `found_amount` is given, then so is `missing_amount`.

So I think you can just remove the if check here and for the type error add a `#type: ignore` comment with an explanation that `missing_amount` should be given if `found_amount` is also given.

Another approach, probably cleaner, is to combine `found_amount` and `missing_amount` into an optional tuple of Fvals. So this way the function argument passing itself will have this rule explicit.